### PR TITLE
Make IPMI tests pretty, fix indenting

### PR DESCRIPTION
**Pull request by rasibley**
I had some ugly indentation, just a few minor udates to clean it up.

### CODE REVIEW

**Comment by rasibley** [comment]
@veruu is this better :-) ?

File: ipmi/stress/driver/runtest.sh [hunk 1]
```diff
@@ -35,16 +35,16 @@ TEST="/kernel/ipmi/stress/driver"
 rlJournalStart
     rlPhaseStartSetup
     # Exit if not ipmi compatible
-        if [ "$(uname -m)" != "ppc64le" ]; then
-            rlRun -l "dmidecode --type 38 > /tmp/dmidecode.log"
+    if [[ $(uname -m) != "ppc64le" ]]; then
+        rlRun -l "dmidecode --type 38 > /tmp/dmidecode.log"
             if grep -i ipmi /tmp/dmidecode.log ; then
```
Comment (veruu) [inline ipmi/stress/driver/runtest.sh]: The condition should be probably moved back to match `rlRun` too

File: ipmi/stress/ipmitool-loop/runtest.sh [hunk 2]
```diff
@@ -45,42 +45,44 @@ rlJournalStart
...
             rlRun "ipmitool sel clear"
             rlRun "ipmitool sel list" 0,1
             rlRun "ipmitool chassis selftest"
             rlRun "ipmitool mc selftest"
             rlRun "ipmitool mc getenables"
             rlRun "ipmitool mc guid"
             rlRun "ipmitool mc getenables system_event_log"
-	fi
+        fi
             rlRun "ipmitool chassis status"
```
Comment (veruu) [inline ipmi/stress/ipmitool-loop/runtest.sh]: These are not inside `if` so they should probably be moved back as well

**Comment by rasibley** [comment]
Thanks @veruu ! I fixed it, any other changes ?

**Pull request merged**